index.d.ts

```diff
@@ -1,5 +1,26 @@
-class PolyK {
-    static Slice(p: number[][][], ax: number, ay: number, bx: number, by: number)
+declare var PolyK: {
```

Comment:
The latest PR I just made actually converts this library into real functions. The `index.d.ts` definition can look something like this now.

```typescript
interface RaycastResult {
    dist: number,
    edge: number,
    norm: {
        x: number,
        y: number
    },
    refl: {
        x: number,
        y: number
    }
}

export function Slice(p: number[], ax: number, ay: number, bx: number, by: number): number[]
export function IsSimple(p: number[]): boolean
export function IsConvex(p: number[]): boolean
export function GetArea(p: number[]): number
export function GetAABB(p: number[]): {}
export function Triangulate(p: number[]): number[]
export function ContainsPoint(p: number[], ax: number, ay: number): boolean
export function Raycast(p: number[],x : number, y: number, dx: number, dy: number): RaycastResult
export function ClosestEdge(p: number[], x: number, y: number): RaycastResult
```